### PR TITLE
Add config flow for LG Netcast

### PR DESCRIPTION
**Pull request by splinter98**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/104913
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/lg_netcast.markdown [hunk 1]
```diff
@@ -15,61 +16,36 @@ ha_integration_type: integration
...
+      - platform: lg_netcast.turn_on
+        entity_id: media_player.lg_netcast_smart_tv
+    action:
+      - service: wake_on_lan.send_magic_packet
+        data:
+          mac: AA-BB-CC-DD-EE-FF
+          broadcast_address: 11.22.33.44
 ```
 
+Any other [actions](/docks/automation/action/) to power on the device can be configured.
```
Comment (Drafteed) [inline source/_integrations/lg_netcast.markdown]: Typo in URL:

```suggestion
Any other [actions](/docs/automation/action/) to power on the device can be configured.
```

**Review by Drafteed — APPROVED** [review]

File: source/_integrations/lg_netcast.markdown [hunk 2]
```diff
@@ -15,61 +16,36 @@ ha_integration_type: integration
...
-  description: Defines an [action](/docs/automation/action/) to turn the TV on.
-  required: false
-  type: string
-{% endconfiguration %}
+## Turn on action
 
-To get the access token for your TV configure the `lg_netcast` platform in Home Assistant without the `access_token`.
-After starting Home Assistant the TV will display the access token on screen.
-Just add the token to your configuration and restart Home Assistant and the media player integration for your LG TV will show up.
+Home Assistant is able to turn on an LG Netcast TV is you specify an action, provided by an {% term integration %} like [HDMI-CEC](/integrations/hdmi_cec/) or [WakeOnLan](/integrations/wake_on_lan/).
```
Comment (c0ffeeca7) [inline source/_integrations/lg_netcast.markdown]:
```suggestion
Home Assistant can turn on an LG Netcast TV if you specify an action provided by an {% term integration %} like [HDMI-CEC](/integrations/hdmi_cec/) or [WakeOnLan](/integrations/wake_on_lan/).
```

File: source/_integrations/lg_netcast.markdown [hunk 3]
```diff
@@ -15,61 +16,36 @@ ha_integration_type: integration
...
 
-<div class='note'>
-  The access token will not change until you factory reset your TV.
-</div>
-
-## Advanced configuration
+You can create an automation from the user interface, from the device create a new automation and select the **Device is requested to turn on** automation.
+Automations can also be created using an automation action:
 
 The example below shows how you can use the `turn_on_action` the [`wake_on_lan` integration](/integrations/wake_on_lan/).
```
Comment (c0ffeeca7) [inline source/_integrations/lg_netcast.markdown]:
```suggestion
The example below shows how you can use the `turn_on_action` with the [`wake_on_lan` integration](/integrations/wake_on_lan/).
```

File: source/_integrations/lg_netcast.markdown [hunk 4]
```diff
@@ -15,61 +16,36 @@ ha_integration_type: integration
...
-After starting Home Assistant the TV will display the access token on screen.
-Just add the token to your configuration and restart Home Assistant and the media player integration for your LG TV will show up.
+Home Assistant is able to turn on an LG Netcast TV is you specify an action, provided by an {% term integration %} like [HDMI-CEC](/integrations/hdmi_cec/) or [WakeOnLan](/integrations/wake_on_lan/).
 
-<div class='note'>
-  The access token will not change until you factory reset your TV.
-</div>
-
-## Advanced configuration
+You can create an automation from the user interface, from the device create a new automation and select the **Device is requested to turn on** automation.
```
Comment (c0ffeeca7) [inline source/_integrations/lg_netcast.markdown]:
```suggestion
1. To create an automation, go to {% my integrations title="**Settings** > **Devices & services**" %} and open the device page. 
2. Under **Automations**, select the + icon to create an automation with that device.
3. In the dialog, select the **Device is requested to turn on** automation.

```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
Small style tweaks

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @splinter98 :+1:

**Pull request merged**